assets/ui/components/ArticleHeadline.jsx

```diff
@@ -3,7 +3,13 @@ import PropTypes from 'prop-types';
 
 export default function ArticleHeadline({item}) {
     return (
-        item.headline && <h2 className='wire-column__preview__headline'>{item.headline}</h2> || null
+        <h2 className='wire-column__preview__headline'>
```

Comment:
in general I wouldn't change the behaviour when we return null if the field is empty, so should be rather:
```
return item.headline ? (
<h2 ...>...</h2>
) : null;
```

Reply:
Done